pkg/views/workspace/info/view.go

```diff
@@ -195,3 +199,14 @@ func getInfoLineGitStatus(key string, status *apiclient.GitStatus) string {
 
 	return output
 }
+
+func getInfoLinePrNumber(PrNumber *int32, status *apiclient.GitStatus) string {
+	if PrNumber != nil {
+		isAnyUncommitedChanges := len(status.FileStatus) > 0
+		isBranchAhead := status.Ahead != nil && *status.Ahead > 0
+		if !(isAnyUncommitedChanges && isBranchAhead) {
```

Comment:
> If a workspace is created from a PR, and the working branch in the workspace has not been changed, the PR number should be displayed in the workspace info section.

This does not refer to any commits made to the branch. It refers to the user not checking out a different branch.

So if the branch from the project repo is the same as the one in the git status, then we display the PR number.

Reply:
sry I misinterpreted the issue. I have pushed a commit with the correct changes now.